spec/models/term_spec.rb

```diff
@@ -138,4 +138,68 @@
       end
     end
   end
+
+  describe "#values_for_property" do
+    context "When requesting a list of values for a property" do
+      let(:label) {RDF::Literal("blah", :language => :en)}
+      let(:resource) do
+        t = Term.new("1")
+        t.label = [label]
+        t
+      end
+
+      it "should return the list of values" do
+        expect(resource.values_for_property(:label)).to eq ["blah"]
+      end
+    end
+
+    context "When requesting a list of values with multiple values" do
+      let(:label1) {RDF::Literal("blah", :language => :en)}
+      let(:label2) {RDF::Literal("banana", :language => :zu)}
+      let(:resource) do
+        t = Term.new("1")
+        t.label = [label1, label2]
+        t
+      end
+
+      it "should return the list of values" do
+        expect(resource.values_for_property(:label)).to eq ["blah", "banana"]
+      end
+    end
+  end
+  describe "#literal_language_list_for_property" do
+    context "When requesting a list of literals with languages for a property" do
```

Comment:
Since this is calling the list of literals and languages there should probably be more than 1 literal and 1 language